docs/technical_reference/unit_models/treatment_models/chem_soft.rst

```diff
@@ -0,0 +1,255 @@
+Chemical Softening
+====================================================
+
+This chemical softening unit model calculates the chemical dose required for target removal of hardness causing components. 
+The model also calculates the size of the mixer, flocculator, sedimentation basin and the recarbonation basin. This chemical softening model:
```

Comment:
I think we need to first say what the model accounts for to provide clarity (i.e., this model includes the following units...)